roles/openshift_node/tasks/config.yml

```diff
@@ -6,6 +6,17 @@
   import_tasks: node_system_container.yml
   when: l_is_node_system_container | bool
 
+- name: Check that node image is present
```

Comment:
Same, we could do this pull first async, then poll for completion to save lots of time.

Reply:
Let's move this above systemd units to get some more tasks between pulling and polling.